TestDrive1/Program.cs

```diff
@@ -9,12 +11,20 @@ static void Main(string[] args)
             // Please keep your IE configuration settings:
             // 1. Check on "Enable Protected Mode" at ALL zones in "Security" tab of Internet Options dialog.
             // 2. Browser zoom level keep to 100%.
-            using (var driver = new OpenQA.Selenium.IE.InternetExplorerDriver())
+            var ieOptions = new InternetExplorerOptions
+            {
+                // Uncomment these lines if you use Microsoft Edge IE mode.
+                // AttachToEdgeChrome = true,
+                // EdgeExecutablePath = "C:/Program Files (x86)/Microsoft/Edge/Application/msedge.exe",
+                IgnoreZoomLevel = true,
+            };
+
+            using (var driver = new OpenQA.Selenium.IE.InternetExplorerDriver(ieOptions))
```

Comment:
`IgnoreZoomLevel = true` should be removed here? If it's OK to set `true`, the above comment (2) is unnecessary.

Reply:
@SIkebe 
> `IgnoreZoomLevel = true` should be removed here?

I would like to keep the code `IgnoreZoomLevel = true`. 
Because the actual zoom level must have been 100%, but the Selenium or WebDriver always reports 175% in my cases.

However, still to be careful to keep the browser's zoom level by 100% even though zoom level is ignored, so I'll keep the comment (2) too.